TrainingApp/SceneDelegate.swift

```diff
@@ -18,8 +18,14 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         window = UIWindow(windowScene: scene)
         
         let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        var rootVC = UIViewController()
+        if(UserDefaults.standard.bool(forKey: R.string.localizable.isuserloggediN()) == true) {
+            rootVC = storyboard.instantiateViewController(withIdentifier: "HomeViewController")
+        }
+        else {
```

Comment:
remove this space
make it
if {
} else {
}

Reply:
Done 👍